server/accounts.go

```diff
@@ -4233,29 +4233,35 @@ func (s *Server) fetch(res AccountResolver, name string, timeout time.Duration)
 	respC := make(chan []byte, 1)
 	accountLookupRequest := fmt.Sprintf(accLookupReqSubj, name)
 	s.mu.Lock()
-	// Resolver will wait for available routes + gateways to reply
+	// Resolver will wait for detected active servers to reply
 	// before serving an error in case there weren't any found.
-	expectedServers := len(s.routes)
-	if s.gateway != nil {
-		expectedServers += len(s.gateway.remotes)
-	}
+	var expectedServers int
 	if s.sys == nil || s.sys.replies == nil {
 		s.mu.Unlock()
 		return _EMPTY_, fmt.Errorf("eventing shut down")
 	}
+	if s.sys != nil && s.sys.servers != nil {
```

Comment:
s.sys check right above, and since you use len(s.sys.servers) it can be nil.

So just need 

```go
expectedServers = len(s.sys.servers)
```

Reply:
good point, just updated this too